### PR TITLE
unit test for getPRequest selector in PullRequestDiscussion

### PR DESCRIPTION
**Pull request by takarzyna**

### CODE REVIEW

File: src/client/containers/PullRequestDiscussion/PullRequestDiscussion.js [hunk 1]
```diff
@@ -31,10 +31,10 @@ export type Props = {
 
 export const getFetchStatus = statusFetchFactory(types.FETCH_PULL_REQUEST_DISCUSSION)
 
-export const getLoggedUser = (state: Object) => state.session.profile.fullName
+export const getLoggedUsername = (state: Object) => state.session.profile.fullName
```
Comment (takarzyna) [inline src/client/containers/PullRequestDiscussion/PullRequestDiscussion.js]: There's another, different `getLoggedUsername` in `ducks/session/selectors`:
`export const getLoggedUsername = state => _.get(state, ['entities', 'me', 'username'], null)`
If they're equivalent, maybe we should replace the one we have here with the one from `ducks/session/selectors`.

Comment (kkateq) [inline src/client/containers/PullRequestDiscussion/PullRequestDiscussion.js]: This is what Williams has fixed in his PR.

File: src/client/containers/PullRequestDiscussion/__tests__/index.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import chai from 'chai'
+import {
+  getPullRequestDiscussion,
+} from '../PullRequestDiscussion'
+import { types } from 'ducks/pullrequests/actions'
+
+const expect = chai.expect
+
+describe('getPullRequestDiscussion', () => {
```
Comment (kkateq) [inline src/client/containers/PullRequestDiscussion/__tests__/index.js]: Please name it properly and add more negative use cases.

File: src/client/containers/PullRequestDiscussion/PullRequestDiscussion.js (hunk 1 above)
Comment (takarzyna) [inline src/client/containers/PullRequestDiscussion/PullRequestDiscussion.js]: If I see correctly, in William's PR the `getPullRequestDiscussion` selector is removed alltogether. Shouldn't we drop my PR then?

**Comment by kkateq** [comment]
@takarzyna #52 has been merged, please proceed on this one

File: src/client/containers/PullRequestDiscussion/PullRequestDiscussion.js (hunk 1 above)
Comment (takarzyna) [inline src/client/containers/PullRequestDiscussion/PullRequestDiscussion.js]: @kkateq done

**Pull request merged**